<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>log_main</class>
 <widget class="QWidget" name="log_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="log_navigation" name="widget" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QStackedWidget { border: 2px solid black; }</string>
     </property>
     <property name="currentIndex">
      <number>5</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="0">
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>注册时间</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>邮箱</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>账户</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>电话</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>70</height>
              </size>
             </property>
             <property name="text">
              <string>修改</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_6">
      <widget class="QToolButton" name="toolButton_5">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>151</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>切换</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QToolButton" name="toolButton_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>111</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>传输</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QToolButton" name="toolButton_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>下载</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QToolButton" name="toolButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>共享</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>log_navigation</class>
   <extends>QWidget</extends>
   <header location="global">log_navigation.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
